### DISCUSSION TITLE
del

### DISCUSSION THREAD

**Discussion by ghost** [bol-van/zapret#356] [General]
@bol-van aka выблядок бляди

**Comment by ghost** [comment]
что я делаю не так?!
запускаю путти, подключился, все ок
вбиваю
opkg update
opkg install iptables-mod-extra iptables-mod-nfqueue iptables-mod-filter iptables-mod-ipopt iptables-mod-conntrack-extra ipset curl ip6tables-mod-nat gzip grep git-http
mkdir /opt
cd /tmp
git clone --depth 1 https://github.com/bol-van/zapret
cd zapret/
./install_easy.sh

do you want the installer to copy it for you (default : N) жму Y
потом еще вопросы, (по советам из сети) жму enter
на вопросе о IPv6 (по советам из сети) жму Y
и далее (по советам из сети) жму enter

все установилось
удаляю zapret из /tmp
cd ..
rm -rf zapret/

иду в /opt/zapret/ipset/zapret-hosts-user.txt
вбиваю заблокированные адерса
после запускаю ipset/get_user.sh
иду в /opt/zapret/ipset/zapret-hosts-users-exclude.txt
вбиваю адреса которые не открываются (dns-shop.ru, citilink.ru, avito.ru, и т.д.)

и ту начинаются пляски, то блокированные сайты работают с пк, на трубке нет, то и на пк, и на трубке нет
то и сайты из zapret-hosts-users-exclude.txt то открываются, то нет

что не так делаю?

**Reply by BubaMuba** [reply]
Указали бы вашу версию OpenWRT. Без этого кто же подскажет.

**Comment by BubaMuba** [comment]
Тогда зачем вы вот это всё устанавливали: opkg install iptables-mod-extra iptables-mod-nfqueue iptables-mod-filter iptables-mod-ipopt iptables-mod-conntrack-extra ipset curl ip6tables-mod-nat?
Прочитайте docs/quick_start.txt и сделайте, как там написано. И везде выбирайте nftables.

**Reply by BubaMuba** [reply]
"Ни хрена не работает" - это малоинформативное сообщение. Напишите, что вы сделали, какие параметры выдал blockcheck, какие параметры задействовали вы.

**Reply by BubaMuba** [reply]
Да вы всё делаете не так)
Для начала надо понимать, что нужно делать, и как это сделать. Автор же специально не давал пошаговых инструкций, чтобы люди включали голову. Я попробую немного вам помочь. Надо сделать так, чтобы в каталоге /tmp оказался подкаталог zapret. То есть, либо вы получаете архив zip, либо используете команду git, но не надо это делать одновременно. Для вас копирую свою инструкцию из другой темы:
1. На PC с линуксом:
~$ git clone --depth 1 https://github.com/bol-van/zapret
2. На PC с линуксом:
~$ scp -r zapret root@192.168.1.1:/tmp/
3. На PC с линуксом:
~$ ssh root@192.168.1.1
4. На железке с openwrt:
~# cd /tmp/zapret
Далее запускайте install_bin.sh, install_prereq.sh и т.д. по инструкции.
Если на железке другой ip, то подставьте его вместо 192.168.1.1

Когда выполните эти четыре пункта, напишите. Если и это не получится, то возможно, стоит поискать специалиста.

**Reply by BubaMuba** [reply]
Ну, зря вы так. Нормальная ОС. То, что я написал, можно сделать и в Windows, но это сложнее.

**Reply by BubaMuba** [reply]
Про автора это уже совсем нехорошо у вас вышло. Умерьте пыл. Сделайте так:
root@RouteRich:/tmp/zapret# ./install_bin.sh
Если не уймётесь с оскорблениями, то это был мой последний совет.

**Reply by BubaMuba** [reply]
Можно что-то не знать. Можно даже много чего не знать. Но кичиться этим, когда оно тебе нужно - это за гранью разумного. И просить совета у людей, оскорбляя их при этом - так только животное может делать.

**Reply by BubaMuba** [reply]
-ash: ./install_bin.sh: Permission denied
Это, вероятно, из-за использования Windows. Сделайте так:
root@RouteRich:/tmp/zapret# chmod +x ./*.sh

**Reply by gram1001** [reply]
> > Про автора это уже совсем нехорошо у вас вышло. Умерьте пыл. Сделайте так: root@RouteRich:/tmp/zapret# ./install_bin.sh Если не уймётесь с оскорблениями, то это был мой последний совет.
> 
> root@RouteRich:/tmp/zapret# ./install_bin.sh -ash: ./install_bin.sh: Permission denied

**su root**  install_bin.sh
